website/src/content/docs/docs/howtos/Generate client libraries/07multipart.mdx

```diff
@@ -50,9 +50,20 @@ def upload(
 ```
 
 ```typescript
-// Not yet implemented. See issues:
-// - RLC: https://github.com/Azure/autorest.typescript/issues/2568
-// - Modular: https://github.com/Azure/autorest.typescript/issues/2422
+// Modular api layer
+export async function upload(
+  context: Client,
+  body: MultipartRequest,
```

Comment:
Could you also provide what does MultipartRequest look like in this case and it's serializer and deserializer in this case?